test/e2e/specs/editor/editor__post-basic-flow.ts

```diff
@@ -206,7 +215,9 @@ describe( DataHelper.createSuiteTitle( 'Editor: Basic Post Flow' ), function ()
 		it( 'Post content is found in published post', async function () {
 			publishedPostPage = new PublishedPostPage( newPage );
 			await publishedPostPage.validateTitle( title );
-			await publishedPostPage.validateTextInPost( quote );
+			for ( const part in quote.split( '\n' ) ) {
+				await publishedPostPage.validateTextInPost( part );
+			}
```

Comment:
Because we use the ParagraphBlock, which types the newlines, the text validation can no longer check for the whole string one at at time. 

Instead, we need to respect the newlines when validating.